clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -1372,4 +1373,8 @@ public PollResult poll(final long currentTimeMs) {
         }
         return PollResult.EMPTY;
     }
+
+    List<MemberStateListener> stateListeners() {
```

Comment:
// for testing only

Reply:
I can add it, but I really do not like those kind of comments.

Reply:
I do not understand why it is so important to state that this method is only used in tests on the implementation class.

Reply:
I'm not saying it's "so important", but it's nice for the reader, no? I won't insist.

Reply:
I do not see big value for the reader. As always, I see the risk that eventually the method is not only used for testing and then we have a lying comment in the code.

Reply:
I agree.